src/app/server/index.js

```diff
@@ -1,16 +1,12 @@
-/* global __DIST__ */
+/* global USE_DEFAULT_KOA_MIDDLEWARES HAS_KOA_MIDDLEWARES KOA_MIDDLEWARES  */
```

Comment:
I commented about this here https://github.com/vgno/roc-web-react/pull/25#discussion_r47008808. 

I believe we need should focus on the ones that are useful for end-users of Roc and less so on internal ones since that is quite common in a Webpack project.